### ISSUE TITLE
SPAWNER material limit not working. 

### ISSUE THREAD

**Issue by iLewisMC** [BentoBoxWorld/Limits#40]
**Description**
Im currently trying to setup 'General Block Limiting' in the config.yml for Limits addons. I have hoppers working fine with a limit of 5 set default and the msg displays when I hit the limit but when im attempting to add 'SPAWNER' as a material it doesnt limit it at all ingame. 
My config currently looks like this;
```
blocklimits:
  HOPPER: 5
  SPAWNER: 3
```

**Steps to reproduce the behavior:**
1. Attempt to limit 'SPAWNER' placement on BSkyblock Islands.

**Expected behavior**
As the config reads above I expected the Limits plugin to limit to a max of 3 spawners and then display a msg (Just as it does with HOPPERS.) 

**Screenshots**
![image](https://user-images.githubusercontent.com/30962238/63927953-9b16e080-ca46-11e9-96c4-b161500b2acc.png)

**Server Information:**

[Please complete the following information:]
 - Database being used YAML
 - OS: Debian 7 Wheezy
 - Java Version: Java 1.8.0_131
 - BentoBox version: 1.6.0 (Same version for Limits)
 - Addons installed?
[11:25:49] [Server thread/INFO]: Running SPIGOT 1.14.4.
[11:25:49] [Server thread/INFO]: BentoBox version: 1.6.0
[11:25:49] [Server thread/INFO]: Database: JSON
[11:25:49] [Server thread/INFO]: Loaded Game Worlds:
[11:25:49] [Server thread/INFO]: bskyblock_world (BSkyBlock): Overworld, Nether, End
[11:25:49] [Server thread/INFO]: Loaded Addons:
[11:25:49] [Server thread/INFO]: BSkyBlock 1.6.0 (ENABLED)
[11:25:49] [Server thread/INFO]: Challenges 0.8.0-SNAPSHOT-#324 (ENABLED)
[11:25:49] [Server thread/INFO]: Level 1.6.0 (ENABLED)
[11:25:49] [Server thread/INFO]: Limits 1.6.0 (ENABLED)
[11:25:49] [Server thread/INFO]: MagicCobblestoneGenerator 1.5.0.1 (ENABLED)
[11:25:49] [Server thread/INFO]: WelcomeWarps 1.6.0 (ENABLED)

[11:29:37] [Server thread/INFO]: Plugins (67): CleanroomGenerator, IronicChest, PlaceholderAPI, MobEggSpawnerBlocker, PlugMan, ChatReaction, OldCombatMechanics, DeluxeTags, MoneyFormatter, ViaVersion, ViaBackwards, WorldEdit, OpenInv, CommandBlocker, ClearLag, MessageAnnouncer, Votifier, ProtocolLib, DeathMessagesPrime, VotingPlugin, MVdWPlaceholderAPI, SketchMapUpdated, DeluxeMenus, PermissionsEx, FeatherBoard, Vault, HolographicDisplays, WorldGuard, Spleef, LockettePro, ChestCommands, ShopGUIPlus, Essentials, BeastWithDraw, BountyHunters, SC-Essentials, BuycraftX, SafeTrade, Citizens, BentoBox, CrazyAuctions, EssentialsChat, ChestShop, NoFlyZone, Quests, EssentialsProtect, Confiscate, EssentialsGeoIP, IslandBorder, ServerSigns, CommandNPC, CombatLogX, EssentialsSpawn, PlayerHeads, TitleManager, Multiverse-Core, Shopkeepers, HungerGames, MineResetLite, BlockParticles, MobArena, KillerMoney, AreaShop, HyperDrive, MineResetLitePlus, ChatControl

**Comment by iLewisMC** [comment]
Adding finding the /bsb limits <player> command I have found out that im limited to 36 spawners. 
![image](https://user-images.githubusercontent.com/30962238/63928737-feedd900-ca47-11e9-93ad-a02815b7ddfc.png)

But no where in the config or my permissions have I stated that value. Only '3' in the config.

**Comment by Poslovitch** [comment]
> But no where in the config or my permissions have I stated that value. Only '3' in the config.

Have you restarted the server upon reducing the value?

**Comment by iLewisMC** [comment]
Yes I did reboot, but to be 100% sure ive just done another reboot and im getting the same results.

**Comment by iLewisMC** [comment]
@tastybento Why arent you considering this issue a bug?

**Comment by tastybento** [comment]
Weird. I missed this for some reason. Please try the latest Limits version. There’s a number of big fixes. Also run a recount on yourself for the island. 
It’s either a bug in the config file, a permission interfering or something else.

**Comment by tastybento** [comment]
**Triage (2026):** This was filed against Limits 1.6.0 / BentoBox 1.6.0 on Spigot 1.14. The block-counting subsystem has since been substantially reworked (per-environment tracking, persistent counts, recount rewrite).

In the current code `SPAWNER` is treated as an ordinary counted block — it is not in the uncountable list and has no special handling in `fixMaterial()`, so a `blocklimits: { SPAWNER: 3 }` entry resolves through the standard limit path like `HOPPER`. The "limited to 36" value the reporter saw never came from the config and was almost certainly a leftover permission or stale stored count rather than a config-read failure.

Recommendation: most likely already resolved. If anyone can still reproduce on a current Limits build, please run a `/<gm> limits calc` recount first and share your `config.yml` + the player's `*.island.limit.*` permissions. Otherwise this can be closed as fixed/stale.

**Comment by tastybento** [comment]
Closing per the triage comment above — a month with no reproduction reported, and the block-counting subsystem has been substantially rewritten since this was filed against Limits 1.6.0. `SPAWNER` resolves through the standard limit path in current code.

If this recurs on a current build: run the admin recount (`/<gamemode admin> limits calc <player>`), then share your `config.yml` and the player's `*.island.limit.*` permissions in a new issue.
